scripts/elasticsearch_nodes.py

```diff
@@ -0,0 +1,123 @@
+"""
+This script is referenced by TerraForm as an external data source.
```

Comment:
Line to short. Rewrap on 80. Both Textmate and Sublime have a function for that.